Makefile

```diff
@@ -0,0 +1,49 @@
+SHELL := /bin/sh
```

Comment:
Let's re-iterate on that, given it won't work on windows.Symfony 4 started using makefiles but then realized it is too much of a huzzle on windows.